README.md

```diff
@@ -26,7 +26,7 @@ dcolgan
 crouchb
 nenns
 jacksonfellows
-jmgrote215
+jmgrote215 jmgrote215 jacksonfellows hutchinsonkeely
```

Comment:
Jackson and Keely's GitHub accounts are jackson-fellows and hutchinson-keely, respectively, my mistake. I assumed they were the same as their KSU username's